README.md

```diff
@@ -0,0 +1,52 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# School Library
+
+> This is a  school library app created with Ruby. The app allows you to store information of teachers, students and the books they borrow.
+
+
+## Built With
+
+- Ruby
+
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Ruby
+- Databases
+### Setup
+- Run `https://github.com/kelomo2502/OOP-school-library.git` in your terminal
+- Switch to the project directory
+- Enjoy ;)
+
+### Install
+- You will need Ruby installed on your system
+
+
+## Author
+
+👤 **Oyewunmi Gbenga Kelvin**
+
+- GitHub: [github](https://github.com/kelomo2502)
+- Twitter: [twitter](https://twitter.com/kelomoJs,)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/oyewunmi-gbenga)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/kelomo2502/OOP-school-library/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] I suggest adding the `MIT.md` file to your project :ok_hand:, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it :wink:
>
> > The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. :100:
>

Reply:
Implemented 👍